galaxy_ng/app/settings.py

```diff
@@ -252,6 +252,12 @@
     SOCIAL_AUTH_KEYCLOAK_EXTRA_DATA = [('refresh_token', 'refresh_token'),
                                        (KEYCLOAK_ROLE_TOKEN_CLAIM, KEYCLOAK_ROLE_TOKEN_CLAIM), ]
 
+    # Enable basic auth using keycloak username/password
+    REST_FRAMEWORK__DEFAULT_AUTHENTICATION_CLASSES = [
+        "rest_framework.authentication.SessionAuthentication",
+        "galaxy_ng.app.auth.keycloak.KeycloakBasicAuth",
+    ]
```

Comment:
when/if #945 is merged, this settings is moved to dynaconf_hooks.py file https://github.com/ansible/galaxy_ng/pull/945/files#diff-51ac604a5e6e00d70270d768fd15bb83bca7f1d919f0471e0f43df8baba9d059R84

Reply:
@newswangerd move this to galaxy_ng/app/dynaconf_hooks.py https://github.com/ansible/galaxy_ng/blob/master/galaxy_ng/app/dynaconf_hooks.py#L84

there you add it to a dictionary
```py
data["REST_FRAMEWORK__DEFAULT_AUTHENTICATION_CLASSES"] = [
    "rest_framework.authentication.SessionAuthentication",
    "galaxy_ng.app.auth.keycloak.KeycloakBasicAuth",
]
```